website/project/views/node.py

```diff
@@ -927,6 +927,7 @@ def _get_summary(node, auth, rescale_ratio, primary=True, link_id=None, show_pat
         'embargo_end_date': node.embargo_end_date.strftime("%A, %b. %d, %Y") if node.embargo_end_date else False,
         'is_pending_embargo': node.is_pending_embargo,
         'archiving': node.archiving,
+        'archive_tree_in_progress': node.root.archiving,
```

Comment:
+1 to this suggestion. This has the additional benefit that `node.root.archiving` will not need to be evaluated if `node.archiving` returns `True`.